### ISSUE TITLE
Syntax error in deploy.sh

### ISSUE THREAD

**Issue by sgibson91** [alan-turing-institute/binderhub-deploy#34]
```
./deploy.sh: line 191: [[: x       1: syntax error in expression (error token is "1")
```

cc @tmbgreaves

**Comment by tmbgreaves** [comment]
I think (hope!) that's a throwback from my previous implementation which was doing string comparison - to avoid the problem of comparing to 'null' I had prepended x to both sides, so there was always a non-empty string to compare. Hopefully removing the xs will fix it...

**Comment by tmbgreaves** [comment]
Suggested fix in PR #22 , https://github.com/alan-turing-institute/binderhub-deploy/pull/22/commits/d1fab6ce08d8c252b26a92b8899db36c2a153427

**Comment by sgibson91** [comment]
Ah!! So the other option would be to have `!=` instead of `-ne`. Sorry, I didn't know if you were doing string comparisons, so just went with the first suggestion my colleague gave me (she did mention string comparison).
